index.js

```diff
@@ -8,8 +8,9 @@ function pixelmatch(img1, img2, output, width, height, options) {
 
     var threshold = options.threshold === undefined ? 0.1 : options.threshold;
 
-    // maximum acceptable square YUV distance between two colors
-    var maxDelta = 255 * 255 * 3 * threshold * threshold,
+    // maximum acceptable square distance between two colors;
+    // 35215 is the maximum possible value for the YIQ difference metric
+    var maxDelta = 35215 * threshold * threshold,
```

Comment:
Ahh, got it. Perfect. It is 35214.74895545784 ≈ 35215 without taking the root. Sorry for the noise!